### PR TITLE
EN-2256 sidebar star filter

### PR DESCRIPTION
**Pull request by gidesan**
https://jira.entando.org/browse/EN-2256

### CODE REVIEW

File: src/ui/digital-exchange/RatingFilterItem.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  const className = `RatingFilterItem${selectedClass}`;
+  const enterKeyCode = 13;
+  const itemLabel = formattedText('digitalExchange.sidebar.ratingFilter.itemLabel');
+  return (
+    <div
+      role="button"
+      tabIndex="0"
+      className={className}
+      onClick={() => onSelect(rating)}
+      onKeyDown={e => e.keyCode === enterKeyCode && onSelect(rating)}
```
Comment (otarza) [inline src/ui/digital-exchange/RatingFilterItem.js]: `keyCode` is deprecated: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

File: src/ui/digital-exchange/RatingFilterContainer.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { connect } from 'react-redux';
+import { convertToQueryString, FILTER_OPERATORS } from '@entando/utils';
+import { fetchDEComponents } from 'state/digital-exchange/components/actions';
+import RatingFilter from './RatingFilter';
```
Comment (otarza) [inline src/ui/digital-exchange/RatingFilterContainer.js]: We use path relative to `src` directory, so this should be 
```import RatingFilterContainer from 'ui/digital-exchange/RatingFilterContainer';```

File: src/ui/digital-exchange/Sidebar.js [hunk 3]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import MarketplaceFilterContainer from 'ui/digital-exchange/MarketplaceFilterContainer';
 import CategoryFilterContainer from 'ui/digital-exchange/CategoryFilterContainer';
+import RatingFilterContainer from './RatingFilterContainer';
```
Comment (otarza) [inline src/ui/digital-exchange/Sidebar.js]: Same here: 
```import RatingFilterContainer from 'ui/digital-exchange/RatingFilterContainer';```

File: src/ui/digital-exchange/RatingFilterItem.js (hunk 1 above)
Comment (gidesan) [reply src/ui/digital-exchange/RatingFilterItem.js]: fixed in commit 94e21ab0189f339d67a26b342b6df9905ab79739

File: src/ui/digital-exchange/RatingFilterContainer.js (hunk 2 above)
Comment (gidesan) [reply src/ui/digital-exchange/RatingFilterContainer.js]: fixed in commit 94e21ab0189f339d67a26b342b6df9905ab79739

File: src/ui/digital-exchange/Sidebar.js (hunk 3 above)
Comment (gidesan) [reply src/ui/digital-exchange/Sidebar.js]: fixed in commit 94e21ab0189f339d67a26b342b6df9905ab79739

**Comment by entando-service** [comment]
All is well

**Pull request merged**